tests/README.md

```diff
@@ -162,37 +162,35 @@ tools/devtool test
 
 ## FAQ
 
-`Q1:` *I have a shell script that runs my tests and I don't want to rewrite it.*
-`A1:` Insofar as it makes sense, you should write it as a python test function.
-However, you can always call the script from a shim python test function. You
-can also add it as a microvm image resource in the s3 bucket (and it will be
-made available under `microvm.slot.path`) or copy it over to a guest filesystem
-as part of your test.
+`Q1:` *I have a shell script that runs my tests and I don't want to rewrite
```

Comment:
I tried lists, and it wouldnt let me do spacing between the bullet points. I found a way to make the raw markdown look a bit nicer by using `\` instead of `<br>` (which won't get reflowed into a single paragraph).